tests/unit_tests/pyanaconda_tests/modules/timezone/test_module_timezone.py

```diff
@@ -587,3 +598,22 @@ def _validate_ntp_config(self, sysroot, was_present, was_enabled, expected_lines
 
         elif not was_present:
             assert not os.path.exists(sysroot + NTP_CONFIG_FILE)
+
+
+class TimezoneHwClockTasksTestCase(unittest.TestCase):
+    """Test the D-Bus Timezone Hardware Clock task."""
+
+    @patch("pyanaconda.modules.timezone.installation.util.execWithRedirect")
+    @patch('pyanaconda.modules.timezone.installation.arch.is_s390', return_value=True)
+    def test_s390_save_hw_clock(self, mock_is_s390, mock_exec_with_redirect):
+        """Test that save_hw_clock does nothing on s390."""
+        self._execute_task(False)
+        # expected state: calling it only once in the check for architecture
+        mock_is_s390.assert_called_once()
+        mock_exec_with_redirect.assert_not_called()
```

Comment:
Could you please add tests that will check the `hwclock --utc` call and `hwclock --local` call? It should be mostly a copy-paste of this test.